sirepo/feature_config.py

```diff
@@ -154,6 +154,11 @@ def b(msg, dev=False):
             "codes that contain proprietary information and authorization to use is granted through oauth",
         ),
         raydata=dict(
+            pdf_temp_dir=(
+                "/home/vagrant/src/radiasoft/raydata/run/",
```

Comment:
How do I do this correctly? I couldn't get it to work without hard coding the whole path

Reply:
For development, you should be relative to srdb.root, which should be setup like it is done in pkcli.service. The default value could be RequiredUnlessDev("run") or something like that. For production, the value would absolutely configured. The configuration parser for pdf_temp_dir would look to see if the directory is relative, then it would be joined with srdbroot. If absolute, it would be used as is. In both cases, the directory must exist(?).

Reply:
ad8603b44b747bc7417cb275935fae525a2aa5e7